constructorio-client/src/main/java/io/constructor/client/ConstructorIO.java

```diff
@@ -1335,4 +1359,129 @@ public String updateCatalog(CatalogRequest req) throws ConstructorException {
             }
         }
     }
+
+    /**
+     * Creates a All Tasks OkHttp request
+     *
+     * @param req the All Tasks request
+     * @return a All Tasks OkHttp request
+     * @throws ConstructorException
+     */
+    protected Request createAllTasksRequest(AllTasksRequest req, String apiVersion) throws ConstructorException {
+        try {
+            List<String> paths = Arrays.asList(apiVersion, "tasks");
+            HttpUrl url = this.makeUrl(paths);
+
+            url = url.newBuilder()
+                    .addQueryParameter("page", String.valueOf(req.getPage()))
+                    .addQueryParameter("num_results_per_page", String.valueOf(req.getResultsPerPage()))
+                    .build();
+
+            Request request = this.makeAuthorizedRequestBuilder()
+                    .url(url)
+                    .get()
+                    .build();
+
+            return request;
+        } catch (Exception exception) {
+            throw new ConstructorException(exception);
+        }
+    }
+
+    /**
+     * Queries the tasks service for all tasks
+     *
+     *
```

Comment:
Extra space that we can probably remove here. Same with the other comments for the function below